Backend/app/rider-platform/rider-app/Main/src/API/UI/Search.hs

```diff
@@ -150,9 +149,9 @@ oneWaySearch personId bundleVersion clientVersion device req = do
   fork "search cabs" . withShortRetry $ do
     becknTaxiReq <- TaxiACL.buildOneWaySearchReq dSearchRes
     void $ CallBPP.search dSearchRes.gatewayUrl becknTaxiReq
-  fork "search metro" . withShortRetry $ do
```

Comment:
why this removed?

Reply:
This we don't use as of now so have commented it out, we can handle the merchant specific Metro later when use case comes. Is t